astropy/config/paths.py

```diff
@@ -52,14 +59,30 @@ def get_config_dir(rootname: str = "astropy") -> str:
         and not (xchpth := xch / rootname).is_symlink()
     ):
         if xchpth.exists():
-            return str(xchpth.resolve())
+            return xchpth.resolve()
         else:
             linkto = xchpth
 
-    return str(_find_or_create_root_dir("config", linkto, rootname))
+    return _find_or_create_root_dir("config", linkto, rootname)
 
 
-def get_cache_dir(rootname: str = "astropy") -> str:
+def get_config_dir(rootname: str = "astropy") -> str:
+    return str(get_config_dir_path(rootname))
+
+
+get_config_dir.__doc__ = (
+    get_config_dir_path.__doc__
+    + """
+    See Also
+    --------
+    get_config_dir_path:
+        same as this function, except that the return value is a pathlib.Path
```

Comment:
According to https://numpydoc.readthedocs.io/en/stable/format.html#see-also examples

```suggestion
    get_config_dir_path : same as this function, except that the return value is a pathlib.Path
```